sw/airborne/firmwares/rotorcraft/stabilization/stabilization_indi.c

```diff
@@ -35,139 +35,149 @@
 
-struct Int32Eulers stab_att_sp_euler;
-struct Int32Quat   stab_att_sp_quat;
-
-static int32_t stabilization_att_indi_cmd[COMMANDS_NB];
-static inline void stabilization_indi_calc_cmd(int32_t indi_commands[], struct Int32Quat *att_err, bool rate_control);
-static void stabilization_indi_second_order_filter_init(struct IndiFilter *filter, float omega, float zeta, float omega_r);
-static void stabilization_indi_second_order_filter(struct IndiFilter *filter, struct FloatRates *input);
-static inline void lms_estimation(void);
-
-#define INDI_EST_SCALE 0.001 //The G values are scaled to avoid numerical problems during the estimation
-struct IndiVariables indi = {
-  .max_rate = STABILIZATION_INDI_MAX_RATE,
-  .attitude_max_yaw_rate = STABILIZATION_INDI_MAX_R,
-
-  .g1 = {STABILIZATION_INDI_G1_P, STABILIZATION_INDI_G1_Q, STABILIZATION_INDI_G1_R},
-  .g2 = STABILIZATION_INDI_G2_R,
-  .reference_acceleration = {
-    STABILIZATION_INDI_REF_ERR_P,
-    STABILIZATION_INDI_REF_ERR_Q,
-    STABILIZATION_INDI_REF_ERR_R,
-    STABILIZATION_INDI_REF_RATE_P,
-    STABILIZATION_INDI_REF_RATE_Q,
-    STABILIZATION_INDI_REF_RATE_R},
-
-  /* Estimation parameters for adaptive INDI */
-  .est = {
-    .g1 = {
-      STABILIZATION_INDI_G1_P / INDI_EST_SCALE,
-      STABILIZATION_INDI_G1_Q / INDI_EST_SCALE,
-      STABILIZATION_INDI_G1_R / INDI_EST_SCALE},
-    .g2 = STABILIZATION_INDI_G2_R / INDI_EST_SCALE,
-    .mu = STABILIZATION_INDI_ADAPTIVE_MU,
-  },
-
-#if STABILIZATION_INDI_USE_ADAPTIVE
-  .adaptive = TRUE,
-#else
-  .adaptive = FALSE,
-#endif
-};
 
 #if PERIODIC_TELEMETRY
 #include "subsystems/datalink/telemetry.h"
 
 static void send_att_indi(struct transport_tx *trans, struct link_device *dev)
 {
-  //The estimated G values are scaled, so scale them back before sending
-  struct FloatRates g1_disp;
-  RATES_SMUL(g1_disp, indi.est.g1, INDI_EST_SCALE);
-  float g2_disp = indi.est.g2 * INDI_EST_SCALE;
-
   pprz_msg_send_STAB_ATTITUDE_INDI(trans, dev, AC_ID,
-                                   &indi.rate.dx.p,
-                                   &indi.rate.dx.q,
-                                   &indi.rate.dx.r,
-                                   &indi.angular_accel_ref.p,
-                                   &indi.angular_accel_ref.q,
-                                   &indi.angular_accel_ref.r,
-                                   &g1_disp.p,
-                                   &g1_disp.q,
-                                   &g1_disp.r,
-                                   &g2_disp);
+                                   &G1[1][0],
+                                   &G1[1][1],
+                                   &G1[1][2],
+                                   &G1[1][3],
+                                   &G1[2][0],
+                                   &G1[2][1],
+                                   &G1[2][2],
+                                   &G1[2][3],
+                                   &G1[3][0],
+                                   &G1[3][1]);
```

Comment:
Maybe better to send an array (and then have user to parse the values) instead of sending individual elements? But indeed this is probably very minor.